test/shared/src/test/scala/zio/test/mock/EnvironmentSpec.scala

```diff
@@ -52,10 +52,10 @@ object EnvironmentSpec extends DefaultRuntime {
 
   def nextInt =
```

Comment:
This test is actually testing for something important so if it isn't passing we need to understand why. When the `MockEnvironment` is created the `MockRandom` is supposed to be seeded with the current live system time (see [here](https://github.com/zio/zio/blob/fd8311581928cbb15ef7275b5364be3a15e9c6f3/test/js/src/main/scala/zio/test/mock/MockEnvironment.scala#L49)) so that when the user runs tests different times they are testing more of the full random "space" instead of just always testing with the same values since the seed is the same. The idea of this test is that if we create two different `MockEnvironment` instances the system time should be at least slightly different so they should have different seeds and produce different values, which has works on JVM and at least the version of JS we have been testing on. Is there something different about node that is causing this not to work? Does the system time always return 0 or something?

Reply:
Due to this: https://github.com/scala-js/scala-js/issues/3623 Scala.js implementation of `System.nanoTime` is limited to stable JS capabilites which `Date` resolution is 1ms. Does it make sense to add 1 ms sleep between these `nextInt` calls then?

Reply:
Interesting. I think this is something we may have to think about more. We test everything on Scalaj.js but not with the Node.js environment, which is why this test is passing in CI but not for you locally. For now yes adding a 1 ms delay makes sense.